scripts/deploy

```diff
@@ -26,30 +26,30 @@ fi
 
 # Setup detailed logging for this entire script
 # In bash 4 it'd be: exec |& tee -a var/log/deploy.details.log
```

Comment:
In older versions of `bash` you can actually do something similar to this entire chunk of spaghetti with:

```bash
exec 2>&1 | tee -a $LOGDETAILS
```

Reply:
I don't understand how this works as-is and the deploy script is annoying to test in not-production so I'm going to leave it alone for now; you're welcome to make this change yourself if it bugs you.